tests/toolchain/gcc5_C_compilation.pm

```diff
@@ -62,6 +65,10 @@ sub test_flags() {
 sub post_fail_hook() {
     my $self = shift;
 
+    script_run('kill %1');
+    script_run('xz /tmp/pidstat.txt');
+    upload_logs('/tmp/pidstat.txt.xz');
+    record_soft_failure 'https://bugzilla.suse.com/show_bug.cgi?id=1024050';
```

Comment:
just write bsc#1024050 and it will get auto-expanded to bsc# but this should also work